src/web/docusaurus/docs/getting-started/environment-setup.md

```diff
@@ -288,7 +294,14 @@ Run the `pnpm db:init` script as it will populate the database with an initial s
 `db:seed` is responsible to populate the database with an initial set of data.
 :::
 
-Stop the `parser` service running. If you have Docker app installed locally make sure to stop the `parser` container before continuing to the next step.
+Stop the `parser` service running. Make sure to stop the `parser` container before continuing to the next step by either using
+
+Docker Desktop app:
+![image](https://user-images.githubusercontent.com/58532267/202260945-30cc22f1-a634-4d4c-9d21-55824fd6bb5a.png)
+
+Docker CLI: `docker stop <container-id>` (e.g, `15649f07c6bf` in above picture)
+
+Switch to app's `.env` by `cp ./config/env.development ./.env` (you changed `.env` previously to seed database).
 
 Finally, restart the `parser` container by running `pnpm:services start parser`.
```

Comment:
For the CLI part, we should decide if we go with `pnpm` scripts or `docker` commands (I vote for `docker` commands). I'm mentioning this because, for stopping the `parser` container, we're recommending  to use a `docker` command (btw, should we mention where to find container ids?), but then for restarting `parser` we have a `pnpm` script.

Also, I don't think we need to copy `config/env.development` to `.env` since the `docker` command to run telescope already includes the proper `env`file:
```sh
 docker-compose --env-file config/env.development up --build -d
```

Reply:
I added the command to find all docker containers `docker container ls -a`. 

I still need to figure out which command we should go for I think 
`docker-compose --env-file config/env.development up --build -d`
requires less copy and pasting and is easier to run which config you want but didn't want more opinions on if i should replace the pnpm commands.

Reply:
@P-DR0ZD you can add the `docker-compose` command as an alternative to `pnpm services:start`